doodad/mode.py

```diff
@@ -284,7 +284,7 @@ def s3_upload(self, file_name, bucket, remote_filename=None, dry=False, check_ex
     def make_timekey(self):
         return '%d'%(int(time.time()*1000))
 
-    def launch_command(self, main_cmd, mount_points=None, dry=False, verbose=False):
+    def launch_command(self, main_cmd, mount_points=None, dry=False, verbose=False, num_exps=1, swap_size=4096):
```

Comment:
Can you add these arguments to the constructor and not the launch_command method? (that way all the modes have the same signature for launch_command)